### DISCUSSION TITLE
Feedback about video player.

### DISCUSSION THREAD

**Discussion by abhinavsp0730** [OpenMined/courses#97] [Feedback] [answered]
Hi, it would be very useful if I'm able to switch the quality of video I want to stream( 480p, 780p, 1080p). And Also it'll Be very useful if there is a button to control the playback speed.
Thank you.

**Comment by yemikifouly** [comment, marked as answer]
Hello Abhinav! 👋 

You can do both of those things. Just scroll down a bit so that you can see the whole player and then click on the setting icon. It is surrounded by a red square on the image below.

![Screenshot_2021-01-03 Course Introduction - Course Outline](https://user-images.githubusercontent.com/13272225/103484239-7de9e300-4ded-11eb-8a5a-3028c2b63ab6.png)

Let me know if this doesn't work for you. 🙂

**Comment by abhinavsp0730** [comment]
Thanks @yemikifouly for your quick response. I was watching the videos in my mobile and in Mobile view these options are hidden.
Thank you I'm closing this issue.

**Reply by abhinavsp0730** [reply]
How to close this issue :-|

**Reply by mcleonard** [reply]
We're going to leave it up so others can see it in the future. But I'll lock it. Thanks!
